src/emscripten_config_vanilla

```diff
@@ -20,7 +20,11 @@ BINARYEN_ROOT = os.path.join(WASM_INSTALL) # directory
 # EMSCRIPTEN_NATIVE_OPTIMIZER='/path/to/custom/optimizer(.exe)'
 
 # See below for notes on which JS engine(s) you need
-NODE_JS = os.path.expanduser(os.getenv('NODE') or '/usr/bin/nodejs') # executable
+prebuilt_node = '{{PREBUILT_NODE}}'
```

Comment:
Duplicated changes are :(

Does it make sense to merge these into `emscripten_config_template`, and add another rewrite step for `{{LLVM_BIN_DIR}}` or so? That seems to be the only thing different between these files